vtex-apps/store-components
# Fix/searchbar component slash

#### What problem is this solving?

This issue is solving when the user is searching for products with slash ( / ) and press enter to search, actually it breaks our url and sometimes we get some not found error, or the search returns a wrong product

#### How to test it?

You can test this modification on dunloppneus account, in iespinoza workspace searching for LT285/75R16 in the searchbar and pressing enter or return key

![image](https://user-images.githubusercontent.com/13649073/91737395-65d72e00-eb85-11ea-83f1-4405cb008fde.png)


If you want to see the older version, can go in the same account in master workspace

[Workspace](https://iespinoza--dunloppneus.myvtex.com/)

#### Screenshots or example usage:


How it was
![image](https://user-images.githubusercontent.com/13649073/91736857-b306d000-eb84-11ea-83f2-55d88ba93155.png)


How it would be
![image](https://user-images.githubusercontent.com/13649073/91736740-86eb4f00-eb84-11ea-9b15-f5acc85d71f4.png)


#### How does this PR make you feel? [:link:](https://giphy.com/gifs/vevo-slash-vevocertified-november-rain-ckohjRxrP1ey4)

![](https://media.giphy.com/media/ckohjRxrP1ey4/giphy.gif)
